File: Java/Program/SoftWare.java
public class SoftWare {
    protected String name;
    protected String version;

    public SoftWare() {}

    public SoftWare(String name, String version) {
        this.name = name;
        this.version = version;
    }

    public void setName(String name) { this.name = name; }
    public void setVersion(String version) { this.version = version; }

    public String getName() { return name; }
    public String getVersion() { return version; }
}

File: CPP/Program/SoftWare.cpp
#pragma once
#include <bits/stdc++.h>
using namespace std;

class SoftWare {
protected:
    string name, version;
public:
    SoftWare() {}
    SoftWare(string n, string v) : name(n), version(v) {}

    void setName(string n) { name = n; }
    void setVersion(string v) { version = v; }

    string getName() { return name; }
    string getVersion() { return version; }
};
